students/mgummel/lesson02/assignment/code/charges_calc.py

```diff
@@ -0,0 +1,160 @@
+'''
+Returns total price paid for individual rentals
+'''
+import argparse
+import json
+import datetime
+import math
+import logging
+import sys
+
+
+LOGGER = logging.getLogger('rental_logger')
+
+
+def parse_cmd_arguments():
+    """
+    Parses all arguments provided to the script at runtime.
+    :return:
+    """
+    parser = argparse.ArgumentParser(description='Process some integers.')
+    parser.add_argument('-i', '--input', help='input JSON file', required=True)
+    parser.add_argument('-o', '--output', help='output JSON file', required=True)
+    parser.add_argument('-d', '--debug', help='debug level 0-3', required=False)
+
+    return parser.parse_args()
+
+
+def set_logging_level(level):
+    """
+    Sets debug level for the program based on the argument passed at runtime.
+    :param level:
+    :return:
+    """
+
+    level_dict = {
+        1: logging.ERROR,
+        2: logging.WARNING,
+        3: logging.DEBUG
+    }
+
+    try:
+        # set debug level for which the logger will capture
+        debug_level = level_dict[int(level)]
+        LOGGER.setLevel(debug_level)
+
+        # create file handler which logs even debug messages
+        log_file = logging.FileHandler('charges_calc.log')
+
+        # create console handler with a higher log level
+        log_stdout = logging.StreamHandler(sys.stdout)
+
+        # create formatter and add it to the handlers
+        formatter = logging.Formatter('%(asctime)s %(filename)s:%(lineno)-3d \
+         %(levelname)s %(message)s')
+        log_stdout.setFormatter(formatter)
+        log_file.setFormatter(formatter)
+
+        if debug_level == logging.DEBUG:
+            log_file.setLevel(logging.WARNING)
+
+        LOGGER.addHandler(log_stdout)
+        LOGGER.addHandler(log_file)
+
+    except KeyError:
```

Comment:
This is a good way to handle an invalid logging level, you should also add a _print_ statement letting the user know they have entered an invalid level.